followthemoney/types/identifier.py

```diff
@@ -1,9 +1,14 @@
 import re
+from typing import Optional, TYPE_CHECKING
+from rigour.ids import FORMATS
```

Comment:
Maybe these could be exposed as a class attribute on the `IdentifierType` class? Then the available formats (and even their docstrings) could be included in the JSON dump of the model, making it possible to include them in the documentation and to pick them up in the TS lib (maybe not to do full validation on the client side, but at least to render some format hints or something like that.)